README.md

```diff
@@ -201,6 +201,9 @@ yarn build:firefox
 Within `apps/namada-interface/`, we may issue the following commands:
 
 ```bash
+# Make sure to build wasm dependencies before starting development server
+yarn wasm:build
+
```

Comment:
I think originally it was assumed that the user already ran `yarn wasm:build` in the previous step (under extension - compiles the same lib that's also needed on interface), in which case they wouldn't need to here, but I think it's perfectly fine to also include it here, as it is required!